### ISSUE TITLE
Models

### ISSUE THREAD

**Issue by BeeGass** [BeeGass/Not-Your-ML-Library#22]
- [ ] Linear Regression
- [ ] Logistic Regression
- [ ] SoftMax Regression

A taxonomy of loss functions should be created and implemented
